grafana-plugin/integration-tests/schedules/quality.test.ts

```diff
@@ -6,7 +6,12 @@ test('check schedule quality for simple 1-user schedule', async ({ page }) => {
   const onCallScheduleName = generateRandomValue();
   await createOnCallSchedule(page, onCallScheduleName);
 
-  await expect(page.locator('div[class*="ScheduleQuality"]')).toHaveText('Quality: Great');
+  /**
+   * this page.reload() call is a hack to temporarily get around this issue
+   * https://github.com/grafana/oncall/issues/1968
+   */
+  await page.reload({ waitUntil: 'networkidle' });
+  await expect(page.locator('div[class*="ScheduleQuality"]')).toHaveText('Quality: Great', { timeout: 15_000 });
```

Comment:
it looks not very reliable to locate elements by className, why not test-id?

Reply:
good catch 👍 I'll add a `data-testid` attribute to that component and refactor this

Reply:
addressed in [this commit](https://github.com/grafana/oncall/pull/1961/commits/267f4e80af079797fa4e3595f09f1fb5f6d4dd7a)